modules/Hoot/ui/sidebar/layerConflate.js

```diff
@@ -212,6 +213,18 @@ class LayerConflate extends SidebarForm {
             delete data.HOOT2_ADV_OPTIONS.RoadEngines;
         }
 
+
+        //If a task grid is present in custom data, use it to restrict conflation
+        let customDataLayer = Hoot.context.layers().layer('data');
+        //Should we also check that a gpx url hash is present?
+        //Or should a manually added (not necessarily TM added)
+        //data layer be availabe for use?
```

Comment:
:point_up: @bobi7 what do you think?

Reply:
I like manually added data layer to be available for use